src/App.tsx

```diff
@@ -1,24 +1,35 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
+/* eslint-disable import/no-cycle */
 import React from 'react';
+import { Footer } from './components/Footer';
```

Comment:
You should add all files to the GitHub
Now all your changes exist only on your computer